models/resource.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+type kvmap map[string]string
+
+// Metadata represents the metadata section of a Kubernetes resource.
+type Metadata struct {
+	Name        string
+	Annotations kvmap `yaml:",omitempty"`
+	Labels      kvmap `yaml:",omitempty"`
+}
+
+// Resource represents the core attributes of a Kubernetes resource. This
+// struct is embedded in all other Kubernetes resource models.
+type Resource struct {
```

Comment:
Everytime I see this I do a double take, could we call this  `MetadataAPIVersionKind` ?

Reply:
Can you expand on why this makes you do a double take? I figured this was the basic information required to identify a Kubernetes resource, so the name seemed appropriate.

Reply:
My reasoning was: 

1. The entire resource is more than it's  MetadataAPIVersionKind (like `spec` for instance, or `users` for groups) 
2. For kustomizations, where `Resource` type is used, you also have `Resources` like [here](https://github.com/larsks/opfcli/blob/main/models/kustomization.go#L12), when I see something like that I think the plural would be a collection/list of the singular, but the 2 aren't really related. 

I don't care that much about this though, we can keep it as is if you prefer.

Reply:
Let's keep it for now; we can always rename it later if we come up with a compelling alternative.